lms/djangoapps/branding/__init__.py

```diff
@@ -17,12 +17,16 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-def get_visible_courses():
+def get_visible_courses(org=None):
     """
     Return the set of CourseDescriptors that should be visible in this branded instance
     """
-    filtered_by_org = microsite.get_value('course_org_filter')
-    courses = CourseOverview.get_all_courses(org=filtered_by_org)
+    microsite_org = microsite.get_value('course_org_filter')
+
+    orgs = [org, microsite_org]
+    orgs = [org for org in orgs if org is not None]
+
+    courses = CourseOverview.get_all_courses(orgs=orgs)
```

Comment:
@jimabramson understood, I hadn't thought of it that way. I agree that it makes the most sense to return an empty result if you try to filter by an invalid org. I'll modify this accordingly and update the tests.